riptable/tests/test_str.py

```diff
@@ -225,33 +264,6 @@ def test_strlen_parallel(self):
         expected = rt.FA([4, 4, 2, 4, 3], dtype=result.dtype).tile(PARALLEL_MULTIPLIER)
         assert_array_equal(result, expected)
 
-    def test_strpbrk_cat(self):

```

Comment:
Would you mind leaving the tests for the older names in place for now? That way we have more confidence the renaming/deprecation mechanism you implemented works correctly and won't break existing code written against the older names.